packages/core/tsconfig.json

```diff
@@ -2,11 +2,14 @@
     "extends": "../../tsconfig.json",
     "compilerOptions": {
         "baseUrl": ".",
-        "rootDir": "src",
+        "rootDir": ".",
```

Comment:
Without this change, the `vendor` files wouldn't properly load, thus throw the error we were seeing.